charts/application-core/templates/_helpers.tpl

```diff
@@ -5,12 +5,29 @@ Expand the name of the chart.
 {{- default .Chart.Name .Values.nameOverride | trunc 63 | trimSuffix "-" }}
 {{- end }}
 
+{{/*
+Build the short stage name based off of the namespace
```

Comment:
Do you want to grab all of the others too? sandbox, demo, integration, performance,..